<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogComp</class>
 <widget class="QDialog" name="DialogComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kомпенсация</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>N</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>NE</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>E</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>SE</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>S</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>SW</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>W</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>NW</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QProgressBar" name="bin0">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QProgressBar" name="bin1">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QProgressBar" name="bin2">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QProgressBar" name="bin4">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QProgressBar" name="bin3">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="bin5">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QProgressBar" name="bin6">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QProgressBar" name="bin7">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
